docs/myst.yml

```diff
@@ -79,6 +79,7 @@ project:
         - file: quickstart-jupyter-lab-myst.md
     - title: Authoring
       children:
+        - file: table-of-contents.md
```

Comment:
I think this should be moved slightly further down the list. Maybe at the bottom?